Jenkinsfile

```diff
@@ -1009,7 +1010,8 @@ pipeline {
                     steps {
                         functionalTest inst_repos: el7_daos_repos(),
                                        inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                                                  ' ' + functional_rpms +
+                                                  ' python2-tabulate '
```

Comment:
Do we need this since we run all cart tests in functional vm?

Reply:
> Do we need this since we run all cart tests in functional vm?

Without the `python2-tabulate` param, we get [missing `tabulate` error](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3139/8/testReport/junit/(root)/TestError/Test___Functional___01___cart_rpc_cart_rpc_two_node_py_CartRpcTwoNodeTest_test_cart_rpc_defaultENV_sep_hosts_1_proto_np_1_a59c/).

Reply:
Which is what people installing the daos-tests RPM will get when they try to run the tests, yes?  They shouldn't then have to install that manually, only after getting the error themselves.  Making this a dependency of daos-tests will mean that when they install daos-tests they also get python2-tabulate and then won't get the error.

Reply:
[python2-tabulate](https://centos.pkgs.org/6/epel-i386/python2-tabulate-0.7.7-1.el6.noarch.rpm.html) is now in [daos.spec](https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec). See 60c8ef19a03e1a7e5aaee2a59e7c2ab60f6d9362.